doc/manpages/man1/qrest.fr.1

```diff
@@ -0,0 +1,30 @@
+.TH man 1 "12 Jan 2013" "0.5" "Qrest"
+.SH NOM
+qrest \- Qrest Really Easy Studio Toolkit
+.SH SYNOPSIS
+qrest
+.SH DESCRIPTION
+.B qrest
+aide à calculer les temps de délai et les fréquences de LFO en fonction du tempo.
+.P
+Il y a 2 façons d'entrer les valeurs de tempo :
+.IP \(bu 2
+Entrée numérique au clavier : l'utilisateur entre simplement la valeur de tempo dans un champ de texte et appuie sur \'entrée\'.
+.IP \(bu 2
+Clic au bouton en temps-réel : l'utilisateur clique sur un bouton en synchronisation avec des pulsations, et qrest lui dira s'il tape régulièrement.
+.P
+Les résultats sont affichés pour toutes les notes comprises entre une ronde et une triple-croche, et un ensemble de boutons-radio permet à l'utilisateur de choisir entre des notes normales, pointées, et des triolets.
+.SH OPTIONS
+Il n'y a actuellement aucune option utilisée.
+.SH BOGUES
+Pas de bogue connu pour le moment.
+.SH REMONTER LES BOGUES
+Utiliser note système de suivi de bogue à http://requests.wardsback.org/browse/QREST
+.SH AUTEUR
+.nf
+Frédéric "deufrai" CORNU (fcornu@wardsback.org)
+.fi
+.SH COPYRIGHT
+Copyright \(co 2008-2012 Frederic CORNU.
```

Comment:
Nope unless you change the mention in the original (EN) file. This is only a translation. So far, I suggest you to keep it this way (ie: 2012) as long as this stay the 0.5 version and change bump both of the FR and EN files when the 0.6 branch becomes the stable release.